libguard/guard_interface.cpp

```diff
@@ -124,6 +124,25 @@ static GuardRecord getHostEndiannessRecord(const GuardRecord& record)
     return convertedRecord;
 }
 
+/**
+ * @brief Function to get the size remaining in the file
```

Comment:
guard file?

Reply:
Yes guard file. You want me to change it to guard file ?

Reply:
yes to avoid confusion

Reply:
Done.